shared/bundle_analysis/report.py

```diff
@@ -142,7 +149,7 @@ def asset_reports(
                 asset_types,
                 chunk_entry,
                 chunk_initial,
-            )
+            ).order_by(ordering(getattr(Asset, ordering_column)))
```

Comment:
Can you explain the purpose of this `getattr` there?... You are grabbing the value from the column listed in `ordering_column`. Is that the same as a reference to the column?

Reply:
It's doing something like `order_by(getattr(Asset, "size"))`, which is really just doing a `order_by(Asset.size)`.